Packs/Securonix/Integrations/Securonix/Securonix.yml

```diff
@@ -393,17 +396,28 @@ script:
     - contextPath: Securonix.ActivityData.Timeline
       description: Time when the activity occurred, in Epoch time.
       type: String
+    - contextPath: Securonix.Activity.totalDocuments
+      description: Total number of events.
+      type: Number
+    - contextPath: Securonix.Activity.message
+      description: Message from the API.
+      type: String
+    - contextPath: Securonix.Activity.queryId
+      description: Query Id for the pagination.
+      type: String
   - arguments:
     - description: Start date/time for which to retrieve activity data (in the format MM/dd/yyyy HH:mm:ss).
       name: from
       required: true
     - description: End date/time for which to retrieve activity data (in the format MM/dd/yyyy HH:mm:ss).
       name: to
       required: true
-    - description: Free-text query, For example, query="resourcegroupname=WindowsSnare and policyname=Possible Privilege Escalation - Self Escalation".
+    - description: "Free-text query, For example, query=\"resourcegroupname=WindowsSnare and policyname=Possible Privilege Escalation - Self Escalation\".\nNote: Use the XSOAR automation browser to run the command if the \"query\" argument contains the special characters."
       name: query
     - description: Paginate next set of results.
       name: query_id
+    - description: "Provide the policy type for retrying if the violations are not found in the initial attempt. The types of policies that can be retried are: \"Land Speed\", \"TIER2\", \"DIRECTIVE\", \"BEACONING\"."
+      name: policy_type
```

Comment:
If it is a closed list, you can use predefined, as for example in line 907.

Reply:
It's not closed list. Right now retries needed for these 4 types, that's why we mentioned but in future it could be added for more policy types.